File: 1295-find-numbers-with-even-number-of-digits/1295-find-numbers-with-even-number-of-digits.java
class Solution {
    public int findNumbers(int[] nums) {
        int c = 0;
        for(int i : nums){
            if(numberOfDigit(i) % 2 == 0)
                c++;
        }
        return c;
    }
    
    public int numberOfDigit(int n){
        int digit = 0;
        
        while(n != 0){
            digit++;
            n /= 10;
        }
        
        return digit;
    }
}

File: 1295-find-numbers-with-even-number-of-digits/1295-find-numbers-with-even-number-of-digits.swift
class Solution {
    func findNumbers(_ nums: [Int]) -> Int {
        var evenNumberOfDigits = 0
        
        for i in nums {
            if isEvenNumberOfDigits(i) {
                evenNumberOfDigits += 1
            }
        }
        
        return evenNumberOfDigits
    }
    
    private func isEvenNumberOfDigits(_ n: Int) -> Bool {
        var count = 0
        var tmp = n
        
        while tmp > 0 {
            count += 1
            tmp /= 10
        }
        
        return count % 2 == 0 ? true : false
    }
}